2025-05-22 03:23:26 [INFO] 백테스트 시작
2025-05-22 03:23:26 [INFO] 설정: Namespace(model_path='C:\\big19\\final_project\\v05_sac-trading\\models\\episode_10_sac_model_20250522_025459', config_path=None, data_path=None, use_db=True, db_user='postgres', db_password='mysecretpassword', db_host='192.168.40.193', db_port=5432, db_name='mydb', symbol='AAPL', results_dir='results/backtest', initial_balance=10000.0, transaction_fee_percent=0.0025, benchmark_data_path=None, window_size=None, override_window_size=False, feature_selection=None)
2025-05-22 03:23:26 [INFO] 설정을 성공적으로 로드했습니다.
2025-05-22 03:23:26 [INFO] 사용할 장치: cuda
2025-05-22 03:23:26 [INFO] 데이터베이스에서 데이터 로드 중...
2025-05-22 03:23:28 [INFO] 데이터베이스에서 AAPL 데이터 로드 완료: 306767 행
2025-05-22 03:23:31 [INFO] 백테스트 실행 중...
2025-05-22 03:23:32 [ERROR] 백테스트 실행 중 오류 발생: 'TradingEnvironment' object has no attribute 'envs'
2025-05-22 03:23:32 [ERROR] Traceback (most recent call last):
  File "C:\big19\final_project\v05_sac-trading\src\backtesting\run_backtest.py", line 453, in main
    results = backtester.run_backtest(verbose=True)
  File "C:\big19\final_project\v05_sac-trading\src\backtesting\backtester.py", line 92, in run_backtest
    next_state, reward, done, info = self.env.step(action)
  File "C:\big19\final_project\v05_sac-trading\src\environment\trading_env.py", line 202, in step
    info = self._get_info()
  File "C:\big19\final_project\v05_sac-trading\src\environment\trading_env.py", line 493, in _get_info
    - sum([env.total_shares_purchased + env.total_shares_sold for env in self.envs.values()]))
AttributeError: 'TradingEnvironment' object has no attribute 'envs'

